pkg/k8shandler/kibana/configmap.go

```diff
@@ -52,12 +44,10 @@ func (clusterRequest *KibanaRequest) createOrUpdateConfigMap(configMap *core.Con
 				return fmt.Errorf("Failed to get %v configmap for %q: %v", configMap.Name, clusterRequest.cluster.Name, err)
 			}
 
-			if checkData {
```

Comment:
This change appears valid but what is "checkData" ?

Reply:
Check L36 above. We used to check data for all config maps except for the TrustedCABundle. The trusted CA bundle logic didn't belong here in the first place. Thus, I dropped the extra check